syntax = "proto3";

package user;

option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1;
    bytes message = 2;
}

message LoginRequest
{
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse
{
    ResultCode result = 1;
    bool success = 2;
}

service UserServiceRpc
{
    rpc Login(LoginRequest) returns(LoginResponse);
}